agent/consul/server_test.go

```diff
@@ -19,12 +19,12 @@ import (
 	"github.com/hashicorp/raft"
 	"google.golang.org/grpc"
 
-	"github.com/hashicorp/consul/agent/rpc/middleware"
-	"github.com/hashicorp/consul/ipaddr"
-
 	"github.com/hashicorp/go-uuid"
 	"golang.org/x/time/rate"
 
+	"github.com/hashicorp/consul/agent/rpc/middleware"
+	"github.com/hashicorp/consul/ipaddr"
+
```

Comment:
Can we put these in the right block below?

Reply:
not sure what happened here 🤔 , thanks for the call out!